EIPS/eip-5065.md

```diff
@@ -0,0 +1,53 @@
+---
+eip: 5065
+title: Instruction for transferring ether
+description: Instruction for just transferring ether without transferring the flow of execution
+author: Mudit Gupta (@maxsam4)
+discussions-to: https://ethereum-magicians.org/t/eip-5065-instruction-for-transferring-ether/9107
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-04-30
+requires: 2929
+---
+
+## Abstract
+Add a new instruction that transfers ether to a destination address without handing over the flow of execution to it. It should work similarly to how `SELFDESTRUCT` transfers ether to the destination without making a call to it.
+
+## Motivation
+From an architectural point of view, execution flow should never be handed over to an untrusted contract. Ethereum currently does not have any ideal way to transfer ether without transferring the flow of execution. People have come up with reentrancy guards and similar solutions to prevent some types of attacks but it's not an ideal solution. The only way to transfer ether from smart contracts without triggering a call is to create a dummy contract, send the precise amount of ether to it and then call `SELFDESTRUCT` from it.
+
+## Specification
+Introduce a new instruction, `AIRDROP` that transfers ether to the destination without making a call to it.
```

Comment:
Please pick an unused opcode number and include it here.  You may want to include the reasoning for your choice of number (if any) in the Rationale.  If it ends up being in conflict by the time this gets closer to release we can update the PR to change it.
```suggestion
Introduce a new instruction, `AIRDROP (0x..)` that transfers ether to the destination without making a call to it.
```

Reply:
Added